### DISCUSSION TITLE
WiX Toolset 5.0.0-rc.1 findings

### DISCUSSION THREAD

**Discussion by ds-andreasor** [wixtoolset/issues#8026] [Questions] [answered]
Hello,
I will start with a discussion and can create issues if needed on monday.

1. The wix repository [https://github.com/wixtoolset/wix](https://github.com/wixtoolset/wix) develop branch is missing a v5.0.0-rc.1 tag and a v5.0.0-rc.1 Pre-release.

2. It is not possible to build with MSBuild and <Project Sdk="WixToolset.Sdk/5.0.0-rc.1"> in the project file because the project system can not find what it needs. Not sure why because the WixToolset.Sdk package on NuGet has a version 5.0.0-rc.1.

Anyone else with this problem?

**Comment by chrpai** [comment]
Yes, I am seeing the same thing. I clearly see 5.0.0-rc.1 on the nuget site but when I try to update from 4.0.4 to 5.0.0-rc.1 in Visual Studio the .wixproj will no longer load.  I looked to see if there was a new version of Heatwave and didn't see one.  I started with a blank Heatwave MSI v4 Package project template just to make sure it wasn't something I was doing wrong in my v4 templates.

![image](https://github.com/wixtoolset/issues/assets/43178814/7f553a47-ccb2-4254-9fa5-7b01859ec4b5)
er lo

**Reply by robmen** [reply]
There is not an update to HeatWave to support v5.0.0-rc.1 yet.

**Comment by robmen** [comment, marked as answer]
1. This is fixed now.
2. Building with MSBuild from the command line works correctly for me.

**Reply by ds-andreasor** [reply]
Thank you,
yes dotnet build does work with my test bundle solution for me.

But some refactorings to use new v5 features are easier with Visual Studio support so I will wait a bit longer.

**Comment by ds-andreasor** [comment]
A general question about the discussions.
Should a discussion be closed if it is answered or should it stay open for later reference to other people?

**Reply by robmen** [reply]
That's a good question. I'm inclined to think discussions should be left open generally but it is _really_ helpful if the answer is marked (as you did). 👏🏻

**Comment by chrpai** [comment]
I can only build if I don't include any extensions.  IIS, Util, NetFx and probably more all reject WiX 5.0.0-rc.1 and any attempt to use their namespaces and elements in the wxs results in a build break in msbuild.   

C:\Users\redacted\.nuget\packages\wixtoolset.sdk\5.0.0-rc.1\tools\wix.targets(581,5): warning : WixToolset.Util.wixext
;WixToolset.UI.wixext;WixToolset.Netfx.wixext;WixToolset.Iis.wixext does not have support for WiX Toolset v5.0.0-rc.1.
[C:\1\Test\Test\Test.wixproj]

@robmen I'm also curious how this was fixed. I don't see any activity in Nuget in the past 3 days. I only see one commit to github to start rc2 and I'm still running HeatWave 1.0.2.

**Reply by ds-andreasor** [reply]
I think the fix was for my first question as it is under releases and tags in the wix repository now.

This sounds like you did not update the `<PackageReference Include="WixToolset.Bal.wixext" Version="5.0.0-rc.1" />` and so on.
I use dotnet build from the commandline to build the bundle or package .wixproj file.
MSBuild from the commandline should also work.

**Reply by chrpai** [reply]
The package references were updated to 5.0.0-rc.1.
